src/Components/Comments.jsx

```diff
@@ -0,0 +1,40 @@
+import { getComments } from "../api";
+import { useEffect, useState } from "react";
+import CommentCard from "./CommentCard";
+
+const Comments = (props) => {
+  const { article_id } = props;
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getComments(article_id)
+      .then((comments) => {
+        setComments(comments);
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        setIsLoading(false);
+        return error;
```

Comment:
You should not be returning an error as this does nothing for the user. You could instead display a message to the user saying the article has no comments yet